src/LegendTextIO.jl

```diff
@@ -19,4 +19,15 @@ using RadiationDetectorSignals: group_by_evtno
 include("util.jl")
 include("geant4_csv.jl")
 
+## .root.hit files
```

Comment:
Please use lower case for source file names, so "root_hist_file.jl" or so in this case.

Reply:
Will do.